sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScan.scala

```diff
@@ -227,21 +271,27 @@ object GpuOrcScan extends Arm {
   def canCast(from: TypeDescription, to: TypeDescription): Boolean = {
     import org.apache.orc.TypeDescription.Category._
     if (!to.getCategory.isPrimitive || !from.getCategory.isPrimitive) {
+      // List, Map, Struct, Union is not primitive
```

Comment:
Why this is added ?

Reply:
I added this comment when I read the code framework, it seems redundant now.